components/brave_wallet_ui/components/desktop/views/crypto/index.tsx

```diff
@@ -158,7 +169,8 @@ const CryptoView = (props: Props) => {
     </>
   ), [
     category,
-    defaultWallet,
+    defaultEthereumWallet,
+    defaultSolanaWallet,
```

Comment:
Can remove `defaultEthereumWallet, defaultSolanaWallet, isMetaMaskInstalled, showDefaultWalletBanner` dependancies here and replace with `showBanner`.

Reply:
Fixed in 5dc125b81e0e2a1284d252bb29dc5654b33fe1ec